src/main/java/uk/gov/justice/digital/delius/config/NationalDatasourceConfig.java

```diff
@@ -37,8 +37,9 @@ public LocalContainerEntityManagerFactoryBean nationalEntityManager(@Qualifier("
         HashMap<String, Object> properties = new HashMap<>();
         properties.put("hibernate.hbm2ddl.auto",
                 env.getProperty("hibernate.hbm2ddl.auto"));
+        final var dialect = env.getProperty("hibernate.dialect");
```

Comment:
Note `getProperty` can take a default value, to save doing the null-check yourself e.g.
```suggestion
        final var dialect = env.getProperty("hibernate.dialect", "org.hibernate.dialect.Oracle12cDialect");
```